patchwork/templates/patchwork/partials/series-list.html

```diff
@@ -0,0 +1,154 @@
+{% load person %}
+{% load listurl %}
+{% load patch %}
+{% load project %}
+{% load static %}
+
+{% include "patchwork/partials/pagination.html" %}
+
+{% if order.editable %}
+<table class="patchlist">
+  <tr>
+    <td class="patchlistreorder">
+      <form method="post" id="reorderform">
+        {% csrf_token %}
+        <input type="hidden" name="form" value="reorderform"/>
+        <input type="hidden" name="order_start" value="0"/>
+        <span id="reorderhelp"></span>
+        <input id="reorder-cancel" type="button" value="Cancel" onClick="order_cancel_click(this)"/>
+        <input id="reorder-change" type="button" value="Change order" onClick="order_button_click(this)"/>
+      </form>
+    </td>
+  </tr>
+</table>
+{% endif %}
+
+{% if page.paginator.long_page and user.is_authenticated %}
+<div class="floaty">
+  <a title="jump to form" href="#patchforms">
+    <span style="font-size: 120%">&#9662;</span>
+  </a>
+</div>
+{% endif %}
+
+<script type="text/javascript">
+$(document).ready(function() {
+    $('#serieslist').stickyTableHeaders();
+
+    $('#serieslist').checkboxes('range', true);
+
+    $('#check-all').change(function(e) {
+        if(this.checked) {
+            $('#serieslist > tbody').checkboxes('check');
+        } else {
+            $('#serieslist > tbody').checkboxes('uncheck');
+        }
+        e.preventDefault();
+    });
+});
+</script>
```

Comment:
Are we doing anything with the checkboxes? Given we don't have any widgets for updating a series, I'd assume not? If so, do we need this?